buildqtc.sh

```diff
@@ -340,6 +340,24 @@ build_unix_qtc() {
 
 	make deployqt
 
+    # Deploy QtWebEngine
+    if [[ $UNAME_SYSTEM == "Linux" ]]; then
+        mkdir -p "$QTC_INSTALL_ROOT/lib/Qt/libexec"
+        cp "$OPT_QTDIR/qtbase/libexec/QtWebEngineProcess" "$QTC_INSTALL_ROOT/lib/Qt/libexec/"
```

Comment:
I thought the jury is still out pondering whether we should use WebEngine or not?

Reply:
yup, but I would anyway commit it and revert with later commits in order to have it preserved in the history - it was not trivial to achieve.

Reply:
Right, and we already have the task in bz to change the documentation. That task can be easily changed to change this instead :)